### ISSUE TITLE
Missing skip link

### ISSUE THREAD

**Issue by sampottinger** [processing/p5.js-web-editor#3084]
### p5.js version

N/A (Online Editor)

### What is your operating system?

Linux

### Web browser and version

Firefox 142

### Actual Behavior

For users not using a pointing technology (for example: navigating and operating by keyboard), the first press on tab takes you to privacy policy.

### Expected Behavior

According to [WCAG success criterion 2.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html), it is common to use a "skip-link" to help users leveraging adaptive technology to reach the body of the application quickly, avoiding nav and other "header" elements. Please consider adding a skip link which jumps the user focus to `#play` or the editor.

### Steps:
1. Open the editor at https://editor.p5js.org/
2. Press tab

**Comment by raclim** [comment]
Thanks for raising this!
